dips/dip-10.md

```diff
@@ -67,19 +77,18 @@ struct DiemIdDomain {
 * Fields definition:
    * `domain` - name of a Diem ID domain 
 * The `DiemIdDomains` resource can only be published into an account with `Role == PARENT_VASP_ROLE_ID`.
-* The `DiemIdDomains` contains a list of `DiemIdDomain` structs that are associated with a VASP. As such, it is possible to register more than one Diem ID Domain for a given VASP. We are providing this opportunity to support possible situations when two VASPs merge into one, or when a company wants to provide multiple wallet apps, while having only a single VASP parent account on the chain.
+* The `DiemIdDomains` contains a list of `DiemIdDomain` structs that are associated with a VASP. As such, it is possible to register more than one Diem ID Domain for a given VASP
 * Only special Treasury Compliance account (address `0xB1E55ED`) can manipulate DiemIdDomains resource:
   * Only TC account can create and publish `DiemIdDomains` resource
   * Only TC account can add, remove or update an `DiemIdDomain` within `DiemIdDomains` resource
-* In order to register a Diem Id domain, a VASP needs to submit a request to Diem Association. LA will perform certain checks (out of scope of this document) before issuing an on-chain transaction to register a Diem Id Domain. These checks intend to mitigate irrelevant claims and enforce uniqueness
-* The Diem ID domain can be created as part of creating a `ParentVASP` account.
-* An entity (potentially the Association) may at some point choose to expose an open source application(*Indexer*) that will allow indexing available Diem IDs and could provide a convenient REST API for applications to fetch information about the domains. The API of such an indexer is out of scope of this RFC.
+* In order to register a Diem ID domain, a VASP needs to submit a request to Diem Association. Diem Association will perform certain checks (out of scope of this document) before issuing an on-chain transaction to register a Diem ID Domain. These checks intend to mitigate irrelevant claims and enforce uniqueness
+* `DiemIDDomains` resource will be created as part of creating a `ParentVASP` account resource, and existing `ParentVASP` accounts without `DiemIDDomains` will have the resource instantiated by the DiemRoot account.
```

Comment:
A couple questions/comments here:
* I think it would be helpful to say what state the `DiemIDDomains` resource will be created in (i.e., empty). Or, does a VASP need to have at least one `DiemIDDomain` allocated to it at the time of creation?
* For existing vasps, they'll need to send a transaction to publish an empty `DiemIDDomains` resource before the association can allocate `DiemIDDomains` to it unless an administration script is used, however that seems pretty heavyweight. Relatedly, should this be the DiemRoot, or TreasuryCompliance account? It seems like it should be TreasuryCompliance to agree with the statement on line 80.